package.json

```diff
@@ -52,14 +52,15 @@
     "jquery": "^3.5.1",
     "json-schema": "0.2.3",
     "lang.js": "^1.1.14",
-    "leader-line": "^1.0.5",
+    "leader-line": "^1.0.7",
     "lodash": "^4.17.19",
     "m-class-list": "^1.1.9",
     "moment": "^2.27.0",
     "node-sass": "^4",
     "plain-draggable": "^2.5.12",
     "pointer-event": "^1.0.0",
     "popper.js": "^1.16.1",
+    "skeleton-loader": "^2.0.0",
```

Comment:
Can you guys give a summary why this is a solution?

Reply:
To my knowledge, the `leader-line` library is quite old and it doesn't support js modules, i.e. it doesn't have `export` in it. So we have to use a `skeleton-loader` to add it.